source/sass/blocks/advantages.scss

```
@@ -32,11 +52,69 @@
 }
 
 .advantages__item {
+  display: flex;
+
   font-family: "Lato", sans-serif;
   font-size: 14px;
   font-weight: 400;
   font-style: normal;
   line-height: 18px;
 
   color: #444444;
+
+  gap: 20px;
+
+  @media (min-width: $tablet-width) {
+    display: block;
+  }
+
+  &::before {
+    content: "";
+
+    display: block;
+    flex-shrink: 0;
+
+    width: 60px;
+    height: 60px;
+
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+}
+
+.advantages__item--useful {
+  &::before {
+    background-color: #68b738;
+    background-image: url("../img/icon_leaf.svg");
+  }
+}
```

```suggestion
.advantages__item {
  display: flex;

  font-family: "Lato", sans-serif;
  font-size: 14px;
  font-weight: 400;
  font-style: normal;
  line-height: 18px;

  color: #444444;

  gap: 20px;

  &::before {
    content: "";

    display: block;
    flex-shrink: 0;

    width: 60px;
    height: 60px;

    background-repeat: no-repeat;
    background-position: center;
  }
  
  &--useful::before {
    background-image: url("../img/icon_leaf.svg");
  }
  
  ....
  
    @media (min-width: $tablet-width) {
    display: block;
  }
}
```

Можно объединить модификаторы с предыдущим селектором через `&` - кода подсократится

И второе - как только импортируешь картинки правильно (без фона) можно будет перенести его в сам `before`, не зависимо от модификаторов